x-pack/plugins/ml/public/jobs/jobs_list_new/jobs.js

```diff
@@ -8,26 +8,57 @@
 import './styles/main.less';
 import { NewJobButton } from './components/new_job_button';
 import { JobsListView } from './components/jobs_list_view';
+import { mlNodesAvailable, permissionToViewMlNodeCount } from 'plugins/ml/ml_nodes_check/check_ml_nodes';
 
 import React, {
   Component
 } from 'react';
 
 import {
   EuiHorizontalRule,
+  EuiCallOut,
+  EuiLink,
+  EuiSpacer,
 } from '@elastic/eui';
 
+function NodeAvailableWarning() {
+  const isCloud = false; // placeholder for future specific cloud functionality
+  if ((mlNodesAvailable() === true) || (permissionToViewMlNodeCount() === false)) {
+    return (<span />);
+  } else {
+    return (
+      <React.Fragment>
+        <EuiCallOut
+          title="No ML Nodes available"
+          color="warning"
+          iconType="alert"
+        >
+          <p>
+            There are no ML nodes available.
```

Comment:
Seeing as we can't have tooltips on disabled buttons, I wonder if we could add a note to the warning saying that you will be unable to create new jobs, or start or stop datafeeds?